Apsitvarkom.UnitTests/DataAccess/PollutedLocationDatabaseRepositoryTests.cs

```diff
@@ -182,18 +167,18 @@ public async Task GetByPropertyAsync_SeveralInstancesWithPropertyFound_FirstMatc
         var dbRows = new List<PollutedLocation>
         {
             new() { Id = Guid.NewGuid() },
-            new() { Id = id, Severity = LocationSeverityLevel.Low },
+            new() { Id = id, Severity = PollutedLocation.SeverityLevel.Low },
             new() { Id = Guid.NewGuid() },
-            new() { Id = Guid.NewGuid(), Severity = LocationSeverityLevel.Low}
+            new() { Id = Guid.NewGuid(), Severity = PollutedLocation.SeverityLevel.Low}
```

Comment:
```suggestion
            new() { Id = Guid.NewGuid(), Severity = PollutedLocation.SeverityLevel.Low }
```
this one's on my previously done changes, but as I've noticed it now, might as well change it here I guess